pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -102,18 +102,15 @@ let
 
       nameArray = builtins.map(a: a.name) (lib.optionals usesNixExtensions nixExtensions);
 
-      requiresSigning = browser ? MOZ_REQUIRE_SIGNING
-                     -> toString browser.MOZ_REQUIRE_SIGNING != "";
-
       # Check that every extension has a unqiue .name attribute
       # and an extid attribute
       extensions = if nameArray != (lib.unique nameArray) then
         throw "Firefox addon name needs to be unique"
-      else if requiresSigning && !lib.hasSuffix "esr" browser.name then
-        throw "Nix addons are only supported without signature enforcement (eg. Firefox ESR)"
+      else if browser.requireSigning || !browser.allowAddonSideload then
+        throw "Nix addons are only supported with signature enforcement disabled and addon sideloading enabled (eg. LibreWolf)"
```

Comment:
Is `Nix addons` a quirky term for `Nixpkgs packaged Firefox extensions`?

Reply:
Probably more accurately a quirky term for "Addons given to Firefox through the wrapper", which by extension is "Firefox extensions packaged with Nix"

Reply:
I think that should be clarified at some point, but not necessarily in this changeset.